src/components/MainSection.js

```diff
@@ -0,0 +1,196 @@
+import React, { useState, useEffect, useRef } from "react";
+import styled from "styled-components";
+import mainImage from "../images/illustration-working.svg";
+import bgBottom from "../images/bg-shorten-mobile.svg";
+import LinkShortener from "./LinkShortener";
+import { pallete } from "../misc/pallete";
+import ShortenedLink from "./ShortenedLink";
+import StatComponent from "./StatComponent";
+import { statList } from "../misc/statList";
+import Button from "./Button";
+import { AnimatePresence } from "framer-motion";
+
+export default function MainSection() {
+  const [linkArr, setLinkArr] = useState([]);
+  const ref = useRef(null);
+  useEffect(() => {
+    const check = localStorage.getItem(`links`);
+    if (check) {
```

Comment:
tu ewentualnie zastanowiłbym się nad nazwą, bo 'check' nie do końca wskazuje na to, czego się tam możemy spodziewać w tym local storage :D

Reply:
w ogóle mega plus za local storage!